Triangulation/test/Triangulation/test_delaunay.cpp

```diff
@@ -112,13 +113,21 @@ void test(const int d, const string & type, const int N)
 template< int D >
 void go(const int N)
 {
-    typedef CGAL::Epick_d<CGAL::Dimension_tag<D> > FK;
-    typedef CGAL::Delaunay_triangulation<FK> Triangulation;
+    typedef CGAL::Epick_d<CGAL::Dimension_tag<D> > KI;
+    typedef CGAL::Delaunay_triangulation<KI> Triangulation;
     test<Triangulation>(D, "static", N);
 
-    typedef CGAL::Epick_d<CGAL::Dynamic_dimension_tag> FK_dyn;
-    typedef CGAL::Delaunay_triangulation<FK_dyn> Triangulation_dyn;
+    typedef CGAL::Epick_d<CGAL::Dynamic_dimension_tag> KI_dyn;
+    typedef CGAL::Delaunay_triangulation<KI_dyn> Triangulation_dyn;
     test<Triangulation_dyn>(D, "dynamic", N);
+
+    typedef CGAL::Epeck_d<CGAL::Dimension_tag<D> > KE;
+    typedef CGAL::Delaunay_triangulation<KE> TriangulationE;
+    test<TriangulationE>(D, "static", N);
+
+    typedef CGAL::Epeck_d<CGAL::Dynamic_dimension_tag> KE_dyn;
+    typedef CGAL::Delaunay_triangulation<KE_dyn> TriangulationE_dyn;
+    test<TriangulationE_dyn>(D, "dynamic", N);
```

Comment:
You forgot `inexact` and `exact`, in the names.

Reply:
Indeed, done. Maybe I started with this file and only thought about adding (in)exact while doing the second file, I don't remember.